packages/kit/src/core/sync/create_manifest_data/index.js

```diff
@@ -267,6 +267,7 @@ export default function create_manifest_data({
 	if (fs.existsSync(config.kit.files.params)) {
 		for (const file of fs.readdirSync(config.kit.files.params)) {
 			const ext = path.extname(file);
+			if (!['.js', '.ts'].includes(ext)) break;
```

Comment:
maybe it should be `continue` here? maybe i'm wrong

Reply:
Oh, I'm wrong!
It should be `continue`.